x-pack/plugins/session_view/server/routes/io_events_route.ts

```diff
@@ -55,9 +55,15 @@ export const getTTYQueryPredicates = async (
 
   if (lastEventHits.length > 0) {
     const lastEvent: ProcessEvent = lastEventHits[0]._source as ProcessEvent;
-    const range = [lastEvent?.process?.entry_leader?.start, lastEvent[TIMESTAMP]];
+    const sessionEnded = lastEvent.event?.action === EventAction.end && lastEvent['@timestamp'];
+    const lastEventTime = lastEvent['@timestamp'];
+    const rangeEnd =
+      sessionEnded && lastEventTime
+        ? parse(lastEventTime)?.add(30, 'second').toISOString()
+        : new Date().toISOString();
+    const range = [lastEvent?.process?.entry_leader?.start, rangeEnd];
     const tty = lastEvent?.process?.entry_leader?.tty;
-    const bootId = lastEvent?.host?.boot?.id;
+    const bootId = lastEvent?.host?.boot?.id || ' ';
```

Comment:
if `lastEvent?.host?.boot?.id` is false, is the default case supposed to be an empty string `''` or is it supposed to be an empty space character `'  '`?

Reply:
Yea.. host.boot.id should be populated by endpoint, but there is a bug there, so i'm short circuiting it until it's fixed. That being said, it's probably better to have this fail over to using host.id as that is actually populated, and should work just as well to query io events for a session. If you are wondering why it's done like this vs just querying by session id, it's explained here https://github.com/elastic/security-team/issues/2644 in the rabbit holes section.